src/main/java/com/twilio/taskrouter/application/CreateWorkspaceTask.java

```diff
@@ -80,15 +89,14 @@ public static void main(String[] args) {
   public static void addWorkersToWorkspace(WorkspaceFacade workspaceFacade,
                                            JsonObject workspaceJsonConfig) {
     JsonArray workersJson = workspaceJsonConfig.getJsonArray("workers");
-    Activity idleActivity = workspaceFacade.findActivityByName("Idle").orElseThrow(() ->
-      new TaskRouterException("The activity 'Idle' was not found. Workers cannot be added"));
+    Activity idleActivity = workspaceFacade.getIdleActivity();
     workersJson.getValuesAs(JsonObject.class).forEach(workerJson -> {
-      Map<String, String> attributes = new HashMap<>();
-      attributes.put("FriendlyName", workerJson.getString("name"));
-      attributes.put("ActivitySid", idleActivity.getSid());
-      attributes.put("Attributes", workerJson.getJsonObject("attributes").toString());
+      Map<String, String> newWorkerParams = new HashMap<>();
```

Comment:
Would you mind using `workerParams` instead of `newWorkerParams` please? IMO `new` in `newWorkerParams` is adding noise.